src/elements/selection-expansion-panel/selection-expansion-panel.ts

```diff
@@ -169,9 +182,22 @@ export class SbbSelectionExpansionPanelElement extends SbbHydrationMixin(LitElem
 
     this._checked = input.checked;
     this._disabled = input.disabled;
+    this._sizeAttributeObserver.observe(input, { attributeFilter: ['size'] });
```

Comment:
would a disconnect be needed before observing? in case of second init or whatever